addon/content/stubGlobals.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+const { XPCOMUtils } = ChromeUtils.import(
+  "resource://gre/modules/XPCOMUtils.jsm"
+);
+
+/* eslint-disable no-unused-vars */
+XPCOMUtils.defineLazyModuleGetters(this, {
+  BrowserSim: "chrome://conversations/content/modules/browserSim.js",
+  Conversation: "chrome://conversations/content/modules/conversation.js",
+  topMail3Pane: "chrome://conversations/content/modules/misc.js",
+});
+/* eslint-enable no-unused-vars */
+
+window.conversationStore = {
```

Comment:
You use `globalThis` in most places, but not here?

Reply:
Oops, missed that, though I'm still figuring out some of these globals. Will check this later.